source/Octopus.Tentacle.Client/Execution/RpcCallExecutor.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using System;
+using System.Reflection.Emit;
```

Comment:
??

Reply:
🤦🏻 
See above i.e. why make a mistake just once when you can do it twice?